median/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"sort"
+)
+
+func median(intsArr []int) float64 {
+	var answer float64
+	sort.Ints(intsArr)
+	log.Println("Sorted list:", intsArr)
+	val := len(intsArr) / 2
+	log.Println("Half value: ", val)
+	if len(intsArr)%2 == 1 {
+		log.Println("%2 == 1")
+		answer = float64(intsArr[val])
+		return answer
+	}
+
+	log.Println("%2 == 0")
+	answer = (float64(intsArr[val-1]) + float64(intsArr[val])) / 2
```

Comment:
Variable answer is redundant, you can just write `return (float64(intsArr[val-1]) + float64(intsArr[val])) / 2`